client/jetpack-cloud/sections/plugin-management/plugins-overview/index.tsx

```diff
@@ -18,12 +19,20 @@ import { setSelectedSiteId } from 'calypso/state/ui/actions';
 import './style.scss';
 
 interface Props {
-	filter: string;
-	search: string;
-	site: string;
+	filter?: string;
+	search?: string;
+	site?: string;
+	pluginSlug?: string;
+	path?: string;
 }
 
-export default function PluginOverview( { filter, search, site }: Props ): ReactElement {
+export default function PluginOverview( {
```

Comment:
Is there any specific reason to use the singular form in the function name here? The file itself is called `plugins-overview`, but we are exporting `PluginOverview`.

I think it works because the default export can be imported with any name, but if you search in the codebase for `PluginOverview` specifically, you might be confused that this component isn't used anywhere.

Reply:
Good catch, @vitozev. Right, it works because of the default export, but it is better to keep the name correct. So, I have made the changes. Thanks.